packages/graphql-language-service-interface/src/getDefinition.js

```diff
@@ -71,20 +71,21 @@ export function getDefinitionQueryResultForDefinitionNode(
 ): DefinitionQueryResult {
   return {
     definitions: [getDefinitionForFragmentDefinition(path, text, definition)],
-    queryRange: [getRange(text, definition.name)],
+    queryRange: definition.name ? [getRange(text, definition.name)] : [],
   };
 }
 
 function getDefinitionForFragmentDefinition(
   path: Uri,
   text: string,
-  definition: FragmentDefinitionNode,
+  definition: FragmentDefinitionNode | OperationDefinitionNode,
 ): Definition {
+  invariant(definition.name, 'Expected ASTNode to have a Name.');
   return {
     path,
     position: getPosition(text, definition.name),
     range: getRange(text, definition),
-    name: definition.name.value,
+    name: definition.name ? definition.name.value : '',
```

Comment:
Do we need this ternary, given the invariant above?

```
const name = definition.name;
invariant(name, 'Expected ASTNode to have a name.');
```

and then I would expect down here below that: `name.value || ''` would work.